src/core/storage/localStorageRecycler.ts

```diff
@@ -0,0 +1,35 @@
+import * as Sentry from '@sentry/browser';
+
+import { RainbowError, logger } from '~/logger';
+
+const OBSOLETE_KEYS = ['rainbow.wagmi'];
+
+/**
+ * Cleanup obsolete storage keys
+ */
+export async function localStorageRecycler(): Promise<void> {
+  try {
+    const storage = await chrome.storage.local.get(OBSOLETE_KEYS);
+    if (Object.keys(storage).length === 0) {
+      logger.debug('No obsolete storage keys found');
+      return;
+    }
+
+    const beforeSize = Object.values(storage).reduce(
+      (acc, value) => acc + JSON.stringify(value).length / 1024,
+      0,
+    );
+
+    await chrome.storage.local.remove(OBSOLETE_KEYS);
+
+    logger.info('Storage cleanup completed', {
+      removedKeys: OBSOLETE_KEYS,
+      freedSpaceKB: beforeSize.toFixed(2),
+    });
+  } catch (error) {
+    Sentry.captureException(error);
```

Comment:
I don't think this line is needed since logger.error should trigger an error including the msg